### PR TITLE
remove old invalid co2 computations

### PR DESCRIPTION
**Pull request by stephanegigandet**
fixes bug #1915

### CODE REVIEW

File: lib/ProductOpener/Food.pm [hunk 1]
```diff
@@ -4547,16 +4547,21 @@ sub compute_carbon_footprint_infocard($) {
 	
 	# Limit to France, as the carbon values from ADEME are intended for France
```
Comment (VaiTon) [inline lib/ProductOpener/Food.pm]: Could you add a comment explaining why are we removing those?

File: lib/ProductOpener/Ingredients.pm [hunk 2]
```diff
@@ -154,6 +154,7 @@ sub compute_carbon_footprint_from_ingredients($) {
 		delete $product_ref->{nutriments}{"carbon-footprint-from-meat-or-fish_product"};
 	}
```
Comment (VaiTon) [inline lib/ProductOpener/Ingredients.pm]: Could you add a comment here too?

File: lib/ProductOpener/Food.pm (hunk 1 above)
Comment (stephanegigandet) [reply lib/ProductOpener/Food.pm]: This function computes those values, and at the start of the function we remove previous values if they exist. New values are computed below.

File: lib/ProductOpener/Ingredients.pm (hunk 2 above)
Comment (stephanegigandet) [reply lib/ProductOpener/Ingredients.pm]: Same thing, we delete old values before computing new ones.

File: lib/ProductOpener/Food.pm (hunk 1 above)
Comment (VaiTon) [reply lib/ProductOpener/Food.pm]: @stephanegigandet I mean in the source code :-)

**Pull request merged**